### PR TITLE
New translation on 1129

### PR DESCRIPTION
**Pull request by HENGRui6**
继续对nn进行翻译

[oneflow.nn.Dropout](https://github.com/Oneflow-Inc/oneflow/blob/9f89dae8be186f7f181fe4ea89b2e1bce11d6b26/python/oneflow/nn/modules/dropout.py#L43)
image

[oneflow.nn.ELU](https://github.com/Oneflow-Inc/oneflow/blob/9f89dae8be186f7f181fe4ea89b2e1bce11d6b26/python/oneflow/nn/modules/activation.py#L222)
image

[oneflow.nn.Embedding](https://github.com/Oneflow-Inc/oneflow/blob/9f89dae8be186f7f181fe4ea89b2e1bce11d6b26/python/oneflow/nn/modules/sparse.py#L24)
image

[oneflow.nn.FakeQuantization](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/fake_quantization.py#L23) 
image

[oneflow.nn.Flatten](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/flatten.py#L22) image

[oneflow.nn.FusedBatchNorm1d](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/batchnorm_fused.py#L115) 
9741641049533_ pic
image

[oneflow.nn.FusedBatchNorm2d](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/batchnorm_fused.py#L195) 
141641049989_ pic
151641050002_ pic

[oneflow.nn.FusedBatchNorm3d](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/batchnorm_fused.py#L273) 
image
image

[oneflow.nn.GELU](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/activation.py#L335)
image

[oneflow.nn.GLU](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/activation.py#L994)
image

[oneflow.nn.GroupNorm](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/normalization.py#L27)
image

[oneflow.nn.Hardsigmoid](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/activation.py#L405)
image

[oneflow.nn.Hardswish](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/activation.py#L648)
image

[oneflow.nn.Hardtanh](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/activation.py#L699)
image

[oneflow.nn.Identity](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/linear.py#L25)
image

[oneflow.nn.InstanceNorm1d](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/instancenorm.py#L60)
image[oneflow.nn.InstanceNorm2d](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/instancenorm.py#L146)
image[oneflow.nn.InstanceNorm3d](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/instancenorm.py#L228)

[oneflow.nn.InstanceNorm2d](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/instancenorm.py#L146)
image
image

[oneflow.nn.InstanceNorm3d](https://github.com/Oneflow-Inc/oneflow/blob/b12bfa7b2c98e032d46d2c693fa0977b4c3425d4/python/oneflow/nn/modules/instancenorm.py#L228)
image
image

### CODE REVIEW

**Comment by HENGRui6** [comment]
这个pr因为nn.functional.conv1d的示例代码一直出现问题无法通过doctest，所以暂时将它删除了，这个翻译可能和当前版本的oneflow有冲突 @ChrisGe137

**Comment by S882li** [comment]
[错别字](https://github.com/Oneflow-Inc/oneflow-api-cn/pull/20/#:~:text=oneflow.nn.GroupNorm)，其他的没啥问题了

**Review by S882li — APPROVED** [review]
